wagtail/contrib/frontend_cache/backends.py

```diff
@@ -199,3 +199,169 @@ def _create_invalidation(self, distribution_id, paths):
                     e.response['Error']['Code'],
                     e.response['Error']['Message']
                 )
+
+
```

Comment:
We've tightened up the flake8 config recently so it won't allow three spaces here anymore, please could you rebase this on the latest master?